package.json

```diff
@@ -12,13 +12,15 @@
     "test:unit": "busted .",
     "test:integration": "node --test --experimental-wasm-memory64 **/*.test.js",
     "test": "yarn test:unit && yarn test:integration",
-    "evolve": "yarn aos:build && node tools/evolve.js"
+    "evolve": "yarn aos:build && node tools/evolve.js",
+    "monitor": "node tools/monitor.js"
```

Comment:
i'm not sure i'd call this `monitor` as it's directly writing to the process. 

it's closer to a `tick` or `backfill`

Reply:
[1cc9df8](https://github.com/ar-io/ar-io-ant-registry-process/pull/16/commits/1cc9df8dd28863d53b0fe033e1f34211cabcd15f)